script/cibuild

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bundle exec jekyll build
+bundle exec htmlproofer ./_site --disable-external
```

Comment:
Is there a build where I can see what happens if `data-proofer-ignore` is not present?